SBSIM_OVERVIEW.md

```diff
@@ -123,7 +123,6 @@ The `environment` module provides the reinforcement learning environment where t
 
             *   `_get_action_spec_and_normalizers(...)`: Defines the action space and normalizers based on the building devices and action configurations.
             *   `_get_observation_spec(...)`: Defines the observation space based on the building devices and observation configurations.
-            *   `_format_action(action, action_names)`: Reformat actions if necessary (extension point for subclasses).
```

Comment:
removing documentation for function that was removed (https://github.com/google/sbsim/pull/32/files#r2070820527). 

we have plans to auto-generate the documentation in the future, which will make these manual documentation edits unnecessary.